### PR TITLE
Feature/images

### PR DESCRIPTION
**Pull request by minsulee2**
Update OG image and favicon

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -12,9 +12,10 @@
     <meta property="og:title" content="Face API" />
     <meta property="og:type" content="website" />
     <meta property="og:description" content="Face recognition API for the browser." />
-    <!-- <meta property="og:image" content="" /> -->
+    <meta property="og:image" content="/ogImage.svg" />
```
Comment (pengin7384) [inline public/index.html:15]: Need png image. OG doesn't support svg format. ([link](https://indieweb.org/The-Open-Graph-protocol#Does_not_support_SVG_images))

Comment (minsulee2) [reply public/index.html]: PTAL!

**Comment by pengin7384** [comment]
ogImage.png가 files changed에서 보이지 않습니다

**Comment by pengin7384** [comment]
LGTM!

**Pull request merged**